src/power_grid_model/validation/validation.py

```diff
@@ -149,13 +150,11 @@ def validate_batch_data(
     for batch, batch_update_data in enumerate(batch_data):
         row_update_data = compatibility_convert_row_columnar_dataset(batch_update_data, None, DatasetType.update)
         assert_valid_data_structure(row_update_data, DatasetType.update)
-        id_errors: list[ValidationError] = list(validate_ids_exist(row_update_data, input_data_copy))
```

Comment:
This was removed instead of renamed @Jerry-Jinfeng-Guo 
Ill check this.

Reply:
Thanks for pointing out!

Reply:
Line revived.